src/platform/qt/ts/mgba-zh_CN.ts

```diff
@@ -3250,7 +3245,7 @@ Game Boy Advance 是任天堂有限公司（Nintendo Co., Ltd.）的注册商标
     <message>
         <location filename="../MapView.cpp" line="107"/>
         <source>Map Addr.</source>
-        <translation>映射地址</translation>
+        <translation>映射地址.</translation>
```

Comment:
Any particular reason for this punctuation? In the English it's to mark that it's an abbreviation, which I assume isn't the case in the translation.

Reply:
There is no need to add this punctuation, I just want to make sure that the translation ends with the same punctuation as the source text.